packages/nimble-components/src/wafer-map/tests/worker-renderer.spec.ts

```diff
@@ -0,0 +1,42 @@
+import type { Remote } from 'comlink';
+import { createMatrixRenderer } from '../modules/worker-renderer';
+import type { MatrixRenderer } from '../../../build/generate-workers/dist/esm/source/matrix-renderer';
+
+describe('MatrixRenderer worker:', () => {
+    let matrixRenderer: Remote<MatrixRenderer> | undefined;
+    let terminate: () => void;
+
+    beforeEach(async () => {
+        const result = await createMatrixRenderer();
+        matrixRenderer = result.matrixRenderer;
+        terminate = result.terminate;
+    });
+
+    afterEach(() => {
+        terminate();
+        matrixRenderer = undefined;
```

Comment:
Lie to the compiler in one spot instead of checking in every spot.

```suggestion
        matrixRenderer = undefined!;
```

Reply:
so evil, did it without hesitation 
![image](https://github.com/ni/nimble/assets/110180309/054bfa50-d498-4be9-b943-3b2b317acd1b)